external/privacy-guides/signal.md

```diff
@@ -30,27 +30,27 @@ How to install Signal in Qubes
 This website cannot guarantee that any PGP key you download from the Internet is authentic.
 Always obtain a trusted key fingerprint via other channels, and always check any key you download against your trusted copy of the fingerprint.
 
-1. (Optional)Create a TemplateVM (Debian 9)
+1. (Optional)Create a TemplateVM (Debian, 9 is used as an example but feel free to use any more updated by changing the 9 to a 10, etc.)
 
        [user@dom0 ~]$ sudo qubes-dom0-update qubes-template-debian-9
 
-2. Open a terminal in Debian 9
+2. Open a terminal in Debian 9 (Or your previously chosen template)
 
        [user@dom0 ~]$ qvm-run -a debian-9 gnome-terminal
        
-3. Use these commands in your terminal
+3. Use these commands in your terminal (If you chose a different distribution, such as buster, substitute that for xenial in the 3rd command)
 
        (Optional)[user@debian-8 ~]$ sudo apt-get install curl
-       [user@debian-8 ~]$ curl -s https://updates.signal.org/desktop/apt/keys.asc | sudo apt-key add -
+       [user@debian-8 ~]$ curl -s -x 127.0.0.1:8082 https://updates.signal.org/desktop/apt/keys.asc | sudo apt-key add -
```

Comment:
> fixing curl command for effectivity in templates

`-x 127.0.0.1:8082` was added because this option is now required in templates?

CC: @marmarek

Reply:
Yes, because it is either that, or temporarily actually connecting the template to your network, and this seems the easier and safer alternative.